### PR TITLE
Double clicking on a line shows source code immediately

### PR DESCRIPTION
**Pull request by Kalmalyzer**
Hi,

first time contributor here.

One usability thing that is annoying to me is: if I want to view the source code associated with a message or a stack frame, I first need to select the message or stack frame, and then perform a double-click on the same entry. (A triple-click would also work.)

This change makes it so that a double-click on a not-already-selected entry will open source code view.
In other words:
single click on entry = move focus to entry
double click on entry = view corresponding source code
... regardless of which element is in focus before.

### CODE REVIEW

**Comment by Kalmalyzer** [comment]
This change, too, is something that I only did to the Editor window. I will look into whether it is something that makes sense for the App window as well. Hold on.

**Comment by Kalmalyzer** [comment]
... nope, App window behaves like this:
single click on message = move selection + show stacktrace
single click on stack frame in stacktrace = move selection

So, no changes are needed to App window to make it consistent.
